src/models/thelio-mega-r1.0/repairs.md

```diff
@@ -0,0 +1,538 @@
+
+### Steps to replace the side fans:
+
+1. Follow the steps above to [remove the top case](#removing-the-top-case) and [remove the inner partition](#removing-the-inner-partition).
+2. Pull the rubber tabs until they come loose from the plastic pins holding the fan in place.
+
+![Side fan rubber mounts](./img/side-fan-mounts.jpg)
+
+3. Pull the end of the fan cable through the opening and release any velcro strips that are holding it in place.
+4. When replacing the fan, hold the rubber tabs in place while pushing the plastic pins in from the opposite side.
+
+![Side fan plastic pins](./img/side-fan-pins.jpg)
+
+## Replacing the Thelio-IO boards:
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 25 minutes  
+**Difficulty:** High <span style="color:red;">●</span>  
+
+The Thelio-IO board handles the front power button, fan control, and 2.5" SATA connectors for the system. If one of the Thelio-IO boards becomes defective, it can be replaced using the instructions below.
+
+1. Follow the steps above to [remove the top case](#removing-the-top-case), [remove the inner partition](#removing-the-inner-partition), [remove the top crossbar](#removing-the-top-crossbar), and [remove the front cover from the 2.5" drive cage](#addingremoving-25-storage-drives) for the Thelio-IO board that is being replaced.
+2. Use the cutouts on the opposite side of the chassis to unplug all cabling from the Thelio-IO board that is being replaced.
+
+![Thelio-IO cabling cutouts](./img/thelio-io-cabling-cutouts.jpg)
+
+3. Unscrew the two screws on the top of the upper drive cage (for the top Thelio-IO board) or the bottom of the lower drive cage (for the bottom Thelio-IO board).
+
+![Thelio-IO drive cage screws](./img/thelio-io-drive-cage-screws.jpg)
+
+4. While pulling the loose side of the drive cage away from the Thelio-IO board, move the board back and out of the drive cage.
+5. Place the new Thelio-IO board into the drive cage and replace the screws and wiring.
+
+## Troubleshooting the power button:
+
+If the front power button doesn't power the machine on or doesn't light up when the system is powered on, try the following troubleshooting steps:
+
+1. Ensure the system powers on normally using the internal power button.
+2. Reseat the front power button to ensure it's making proper contact.
+3. Check the wiring for the front power button.
+4. Replace the front power button, if necessary.
+
+**Tools required:** Cross-head (Phillips) screwdriver (optional)  
+**Time estimate:** 20 minutes  
+**Difficulty:** Medium <span style="color:orange;">●</span>  
+
+### Steps to power the machine on using the internal power button:
+
+1. Follow the steps above to [remove the top case](#removing-the-top-case) and [remove the inner partition](#removing-the-inner-partition).
+2. Ensure the system is plugged into power, and the power supply switch is in the 1 (On) position.
+3. Push the small button labeled B0 on the top Thelio-IO board.
```

Comment:
Would it also be good to point out the power button directly on the motherboard? It's definitely easier to access, in my opinion.

Reply:
Checking the Thelio-IO button tells us whether the Thelio-IO board is connected to the motherboard properly or not. I typically start with that since it quickly narrows down the issue to the power button itself. I've added more details and mentioned the motherboard button as well.